libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/compound/CompoundColorPalette.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.designsystem.theme.compound
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import io.element.android.libraries.designsystem.Black_800
+import io.element.android.libraries.designsystem.Black_900
+import io.element.android.libraries.designsystem.Black_950
+import io.element.android.libraries.designsystem.Gray_100
+import io.element.android.libraries.designsystem.Gray_150
+import io.element.android.libraries.designsystem.Gray_200
+import io.element.android.libraries.designsystem.Gray_25
+import io.element.android.libraries.designsystem.Gray_250
+import io.element.android.libraries.designsystem.Gray_300
+import io.element.android.libraries.designsystem.Gray_400
+import io.element.android.libraries.designsystem.Gray_450
+import io.element.android.libraries.designsystem.Gray_50
+
+interface CompoundColorPalette {
+    interface Content {
+        val primary: Color
+        val secondary: Color
+        val tertiary: Color
+        val quaternary: Color
```

Comment:
We should not have things like that as we agreed to go for a material3 theme on android with the design team...
Can you check with them?

Reply:
According to the design team we're still using colors from the old app (both palette and semantic ones) so that's probably why the quaternary and quinary colors still exist. Those will be replaced in the future with the proper material colors, once the new design tokens are ready.

Reply:
I mean, with this extra class we can take a look at the Figma designs that have names such as `Archive/Compound/Light/Quinary Content` and use those. Then, when we need to migrate to the new Material theme we just need to search for references to this class and replace them with the actual Material theme semantic colors.